tests/spin/test_spin_hamiltonian.py

```diff
@@ -401,7 +401,7 @@ def test_heisenberg_hamiltonian(shape, n_cells, j, expected_ham):
         ),
         (
             "square",
-            [2, 2, 1],
+            [2, 2],
```

Comment:
[2, 2, 1] is equivalent to [2, 2]?

Reply:
This was when we were letting the users pass larger dimensions for n_cells, now when we added the error statement, it doesn't work the same way.